google_photos_takeout_helper/__main__.py

```diff
@@ -109,16 +119,16 @@ def for_all_files_recursive(
       folder_function=lambda fo: True,
       filter_fun=lambda file: True
     ):
-        for file in dir.rglob("*"):
+        for file in tqdm(dir.rglob("*")):
```

Comment:
Since `dir.rglob()` returns an iterator `tqdm` has no way of knowing the number of iterations. So we can either leave it as it is, convert the iterator into a list so tqdm can know the number of iterations or pass the `total` argument in case we know the total value.

Reply:
tqdm will track progress and show iterations but cannot know the total number expected.